docs/cli-guide.rst

```diff
@@ -0,0 +1,389 @@
+  in).
+
+- Alternatively, if your plain text file does not need to be modified, you can
+  run ``align`` directly and use the ``-i`` option to indicate that the input
+  is plain text and not xml. You'll also need the ``-l <language>`` option to
+  indicate what language your text is in.
+
+Two additional commands are sometimes useful: ``readalongs tokenize`` and
+``readalongs g2p``.
+
+- ``tokenize`` takes the output of ``prepare`` and tokenizes it, wrapping each
+  word in the text in a ``<w>`` element.
+
+- ``g2p`` takes the output of ``tokenize`` and mapping each word to its
+  phonetic transcription using the g2p library. The phonetic transcription is
+  represented using the ARPABET phonetic codes and are added in the ``ARPABET``
+  attribute to each ``<w>`` element.
+
+The result of ``tokenize`` or ``g2p`` can be fixed manually if necessary and
+then used as input to ``align``.
+
+Getting from TXT to XML with readalongs prepare
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run :ref:`cli-prepare` to prepare an XML file for ``align`` from a TXT file.
+
+``readalongs prepare [options] [story.txt] [story.xml]``
+
+``[story.txt]``: path to the plain text input file (TXT)
+
+``[story.xml]``: Path to the XML output file
+
+The plain text file must be plain text encoded in ``UTF-8`` with one
+sentence per line. Paragraph breaks are marked by a blank line, and page
+breaks are marked by two blank lines.
+
++-----------------------------------+-----------------------------------+
+| Key Options                       | Option descriptions               |
++===================================+===================================+
+| ``-l, --language`` (required)     | The language code for story.txt.  |
++-----------------------------------+-----------------------------------+
+| ``-f, --force-overwrite``         | Force overwrite output files      |
+|                                   | (handy if you’re troubleshooting  |
+|                                   | and will be aligning repeatedly)  |
++-----------------------------------+-----------------------------------+
+| ``-h, --help``                    | Displays CLI guide for            |
+|                                   | ``prepare``                       |
++-----------------------------------+-----------------------------------+
+
+The ``-l, --language`` argument requires a language’s 3 character `ISO
+code <https://en.wikipedia.org/wiki/ISO_639-3>`__ as an argument.
+
+The languages supported by RAS can be listed by running ``readalongs prepare -h``
+and they can also be found in the :ref:`cli-prepare` reference.
+
+So, a full command for a story in Algonquin would be something like:
+
+``readalongs prepare -l alq Studio/story.txt Studio/story.xml``
+
+The generated XML will be parsed in to sentences. At this stage you can
+edit the XML to have any modifications, such as adding ``do-not-align``
+as an attribute of any element in your XML.
+
+.. _dna:
```

Comment:
I'm not sure this should be in the CLI section - it's more of a text/xml/markup editing activity (along with anchors and other things. Maybe that should have its own section?

Reply:
Hum, it's here because it's part of the story told in the guide, of why you might want to run prepare. The steps suggested are 1. run prepare 2. mark you DNAs, 3. run align, and the file tries to follow that structure. But you're right, a section with dna, anchors, silences, etc, might make more sense.